test/suites-ostree/backup-prune.robot

```diff
@@ -20,22 +20,10 @@ ${UNKOWN_DIR_REG}       unknown_fake*
 
 
 *** Test Cases ***
-Rebooting Healthy System Should Result In Pruning Old Backups
-    [Documentation]    Check that MicroShift backup logic prunes old backups
+Rebooting Healthy System Should Result In Pruning Old Backups And Keeping Unknown Directories
+    [Documentation]    Check that MicroShift backup logic prunes old backups and keeps unknown directories
 
-    Create Fake Backup Directories
-    Validate Fake Backups Exist
-    Reboot MicroShift Host
-    Wait For MicroShift Service
     Validate Fake Backups Have Been Deleted
-
-Rebooting Healthy System Should Not Delete Unknown Directories
-    [Documentation]    Check that MicroShift backup logic does not delete unknown directories
-
-    Create Fake Unknown Backup Directories
-    Validate Unknown Fake Backups Exist
-    Reboot MicroShift Host
-    Wait For MicroShift Service
     Validate Unknown Fake Backups Exist
```

Comment:
I may be having some trouble with the naming of these keywords, but why would we expect any unknown backups to exist at this point?

Reply:
Fair, maybe we could reword that, this is validating our pruning logic when it does not delete directories it does not know (follow our backup directory naming convention), so maybe a more accurate wording would be "Unknown Directories". So if for some reason a user places a directory in there or we decide to place something else in there in the future, it does not get deleted by our pruning logic.